jdtls.ext/com.shengchen.checkstyle.runner/src/main/java/com/shengchen/checkstyle/runner/handler/DelegateCommandHandler.java

```diff
@@ -22,23 +22,24 @@
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.jdt.ls.core.internal.IDelegateCommandHandler;
 
+import java.lang.reflect.Method;
 import java.util.List;
 
 @SuppressWarnings("restriction")
 public class DelegateCommandHandler implements IDelegateCommandHandler {
 
-    private static final String CHECK_CODE_WITH_CHECKSTYLE = "java.checkstyle.checkcode";
-    private static final String FIX_CHECKSTYLE_VIOLATION = "java.checkstyle.quickfix";
-
+    private static final String CHECKSTYLE_PREFIX = "java.checkstyle.";
+    
     @Override
     public Object executeCommand(String commandId, List<Object> arguments, IProgressMonitor monitor) throws Exception {
-        switch (commandId) {
-            case CHECK_CODE_WITH_CHECKSTYLE:
-                return CheckstyleRunner.check(arguments);
-            case FIX_CHECKSTYLE_VIOLATION:
-                return CheckstyleRunner.quickFix(arguments);
-        }
+        if (commandId.startsWith(CHECKSTYLE_PREFIX)) {
```

Comment:
Add comments to illustrate the reflect usage here: like, How to call the method, the naming convention.